hr_expense_analytic_require/data/res_company_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="analytic_main_company" model="account.analytic.account">
+            <field name="name">Main Analytic Account</field>
+        </record>
+
+        <record id="base.main_company" model="res.company">
+            <field name="default_expense_analytic_id" ref="analytic_main_company"/>
+        </record>
+    </data>
+</odoo>
```

Comment:
IMO it's demo data. 
Please, explain in USAGE file that is needed to define `default_expense_analytic_id` field in company.

Reply:
This is default setup data. Not adding it adds significant complexity to the implementation.
The documentation on it is in the README in the Configuration section.